README.md

```diff
@@ -104,6 +104,91 @@ pull(
 )
 ```
 
+### `fullPath`
+
+If specified the exporter will emit an entry for every path component encountered.
+
+```javascript
+const exporter = require('ipfs-unixfs-exporter')
+const pull = require('pull-stream')
+const collect = require('pull-stream/sinks/collect')
+
+pull(
+  exporter('QmFoo.../bar/baz.txt', ipld, {
```

Comment:
For the sake of greater clarity, can we `s/ipld/graphService`? //cc @vmx

Reply:
I would go with `ipld` for now until there's really something other than IPLD.